homework/week-2/jorgereina1986/Horoscope/app/src/main/java/jorgereina1986/c4q/nyc/horoscope/SearchActivity.java

```diff
@@ -0,0 +1,84 @@
+package jorgereina1986.c4q.nyc.horoscope;
+
+import android.content.Intent;
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.TextView;
+
+import java.util.Date;
+
+
+public class SearchActivity extends ActionBarActivity {
```

Comment:
I think this could have a more apt name, with greater specificity. Something like FindSign, for example.